src/test/resources/integration/pipelines/declarative/buildAppend.pipeline

```diff
@@ -10,7 +10,7 @@ node("TestSlave") {
     stage "Configuration"
     rtServer(
             id: serverId,
-            url: "${env.JENKINS_ARTIFACTORY_URL}",
+            url: "${env.JENKINS_PLATFORM_URL}".replaceFirst('/*$', '')+'/artifactory',
```

Comment:
Replacing the first slash in the URL means the URL must not include slashes.
We should remove the last character from the URL, if it is a slash. It is best to do this from outside the pipeline script, from within the tests infra code.

Reply:
OK - please ignore. LGTM.